src/PrestaShopBundle/Form/Admin/Sell/Supplier/SupplierType.php

```diff
@@ -108,15 +115,36 @@ public function __construct(
         $this->statesChoiceProvider = $statesChoiceProvider;
         $this->contextCountryId = $contextCountryId;
         $this->isMultistoreEnabled = $isMultistoreEnabled;
+        $this->router = $router;
     }
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $data = $builder->getData();
         $countryId = 0 !== $data['id_country'] ? $data['id_country'] : $this->contextCountryId;
 
+        $invalidCharsText = sprintf(
+            '%s <>;=#{}',
+            $this->trans('Invalid characters:', 'Admin.Notifications.Info')
+        );
+
+        $invalidGenericNameHint = sprintf(
+            '%s <>={}',
```

Comment:
Maybe you should use `TypedRegexValidator::GENERIC_NAME_CHARS` here? Although if we use these const in multiple places maybe they should be in a dedicated settings class instead of `TypedRegexValidator` which is linked to a form constraint